interface/forms/newpatient/templates/newpatient/common.html.twig

```diff
@@ -15,6 +15,13 @@
 #}
 <!DOCTYPE html>
 <html>
+<script>
+    function showSuccess(){
```

Comment:
You may not be aware but all the scripts for this form should go in the newpatient.js file.  You should add the onSubmit via the javascript addEventListener piece.
https://github.com/openemr/openemr/blob/3a1343e3861caf760d199e8385f17f26419d5d13/interface/forms/newpatient/newpatient.js#L54-L53

We did a lot of work to take the javascript stuff out of the twig file and we'd like to keep it that way.

You'l also want to use window.top.xlj in order to get language translation in the javascript.  

Finally bump up the javascript asset version when you make that change so that people get the latest js file.
https://github.com/openemr/openemr/blob/08ed79b7bc8f60f7dec52fa9fb9d93240a5c2a39/version.php#L50

Thank you for the PR and feel free to ask questions if you need help.